lib/trino-errorprone/pom.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <!-- trino-root can't be the parent because it depends on this module -->
+        <groupId>io.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>116</version>
+    </parent>
+
+    <groupId>io.trino</groupId>
+    <artifactId>trino-errorprone</artifactId>
+    <name>trino-errorprone</name>
+    <version>368-SNAPSHOT</version>
+
+    <properties>
+        <project.build.targetJdk>11</project.build.targetJdk>
+
+        <air.main.basedir>${project.basedir}</air.main.basedir>
+        <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
+        <air.check.skip-pmd>true</air.check.skip-pmd>
+        <air.check.skip-jacoco>true</air.check.skip-jacoco>
+
+        <!-- Remember to keep it in sync with trino-root: -->
+        <dep.errorprone.version>2.10.0</dep.errorprone.version>
```

Comment:
"Remember" doesn't work, especially when root pom doesn't link here.

Reply:
Why would we redefine it here?